packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -72,7 +72,12 @@ class Question {
 		text: ''
 	};
 
-	constructor({ title, question, description, viewFolder, fieldName, validators } = {}) {
+	/**
+	 * @type {string} alternative url slug
+	 */
+	url;
+
+	constructor({ title, question, description, viewFolder, fieldName, validators, url } = {}) {
```

Comment:
As this is just the base class, I think the additional url parameter will need to be included in the constructors of the descendent question types and passed through in the call to `super()`.